Lib/glyphsLib/classes.py

```diff
@@ -2117,14 +2119,13 @@ def shouldWriteValueForKey(self, key):
 class GSAnchor(GSBase):
     _classesForName = {"name": unicode, "position": Point}
     _parent = None
-    _defaultsForName = {"position": Point(0, 0)}
+    _defaultsForName = {"position": Point(x=0, y=0)}
 
     def __init__(self, name=None, position=None):
-        super(GSAnchor, self).__init__()
```

Comment:
Why are you removing the super() calls?

Reply:
The super() calls are only needed to pre-populate the instance variables. But maybe `__slots__` is a better mechanism.